css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
```

Comment:
You're applying this styling to the entire app, to all pages, not only on login.
Don't change global styling unless it needs to be applied to all pages.

Reply:
You added a styling to the body tag, this tag is global for the entire site, all pages on this site will have the same tag, if there is nothing specific about it, any styling you add here will reflect on the entire site.
You need to set a custom class for the body tag and apply your styling to it.